### DISCUSSION TITLE
ID deviant TF motifs question

### DISCUSSION THREAD

**Discussion by pseudacriscrucifer** [GreenleafLab/ArchR#2095] [Questions / Documentation]
Hi,

Anyone know what may cause this error? Following along with tutoriat @ 15.4.1...

`rowData(seZ)$maxDelta <- lapply(seq_len(ncol(seZ)), function(x){
  rowMaxs(assay(seZ) - assay(seZ)[,x])
}) %>% Reduce("cbind", .) %>% rowMaxs`

This returns:

`Error in h(simpleError(msg, call)) : 
  error in evaluating the argument 'x' in selecting a method for function 'rowMaxs': error in evaluating the argument 'x' in selecting a method for function 'Reduce': useNames = NA is defunct. Instead, specify either useNames = TRUE or useNames = FALSE.
`

As far as I can tell I have followed along correctly.

Thanks

**Comment by jtls0n** [comment]
try adding useNames = FALSE or useNames = TRUE to the rowMaxs function
